packages/react/src/autocomplete-empty/AutocompleteEmpty.tsx

```diff
@@ -2,24 +2,25 @@ import { type ComponentPropsWithRef, forwardRef } from "react";
 
 import { Box } from "../box";
 
-type CommandProps = ComponentPropsWithRef<typeof Box>;
+type AutocompleteEmptyProps = ComponentPropsWithRef<typeof Box>;
 
-export const AutocompleteEmpty = forwardRef<HTMLDivElement, CommandProps>(
-  ({ children, ...props }, ref) => {
-    return (
-      <Box
-        color="fg.disabled"
-        display="flex"
-        fontSize="lg"
-        justifyContent="center"
-        p="md"
-        ref={ref}
-        {...props}
-      >
-        {children || "No results found"}
-      </Box>
-    );
-  },
-);
+export const AutocompleteEmpty = forwardRef<
```

Comment:
i'm wondering if we should call this `AutocompleteEmptyItem`

so under this umbrella you have `Item` and `EmptyItem`